### PR TITLE
Feature/known skills

### LINKED ISSUE #12: FE: Show the list of yet to be completed skills for each specific dog

### PR DESCRIPTION
**Pull request by Jlavarine**
## Description 
Add functionality of adding skills to dog known skills after completing training.
#### Where to start?
Where should the reviewer start?

#### How to test?
In the project directory, you can run:

### `npm start`
Runs the app in the development mode.\
Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
The page will reload when you make changes.

#### Issues
Closes #12 

#### Type of Change:
- [] Bug fix (non-breaking change which fixes an issue)
- [] Refactor (code is working and without bugs, formatting has been streamlined)
- [x] New feature (non-breaking change which adds functionality)
- [] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### How Has This Been Tested?
What steps have you taken to ensure the code is effective?
- [x] Have you used Dev Tools/ ran code in the console?
- [] Have you ran the Cypress testing suite?

#### Checklist:
- [x] My code follows the style guidelines of this project.
- [x] My commit messages are consistent, descriptive, concise and begin with a verb and capital letter.
- [x] I have performed a self-review of my own code.
- [x] I have added reviewers.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -17,6 +17,7 @@ const App = () => {
   const [username, setUsername] = useState('') 
   const [email, setEmail] = useState('')
   const [dogSkills, setDogSkills] = useState('')
+  const [dogId, setDogId] = useState(0)
```
Comment (aominhlong) [inline src/App.js:21]: Fix spacing

File: src/App.js [hunk 2]
```diff
@@ -140,7 +167,11 @@ const App = () => {
         username: username,
         email: email
       }
-    }).then(data => console.log(data))
+    }).then(data => {
+      console.log(data)
+      setUsername('')
```
Comment (DanielN88) [inline src/App.js:172]: Good catch on this odd bug preventing the homepage from rendering after creating a new user.

File: src/App.js [hunk 3]
```diff
@@ -140,7 +167,11 @@ const App = () => {
         username: username,
         email: email
       }
-    }).then(data => console.log(data))
+    }).then(data => {
+      console.log(data)
```
Comment (aominhlong) [inline src/App.js:171]: Remove console.log(s) if we are not needing them anymore

**Review by DanielN88 — APPROVED** [review]
Everything look good and is ready to merge.

**Comment by aominhlong** [comment]
Great job catching the create user bug!

**Review by aominhlong — APPROVED** [review]

**Review by JoshMallery — APPROVED** [review]
Solid thought process from everyone on making our way through this and meeting the MVP.

**Pull request merged**